cmd/xl-v1-bucket.go

```diff
@@ -294,6 +294,11 @@ func (xl xlObjects) IsNotificationSupported() bool {
 	return true
 }
 
+// IsListenBucketSupported returns whether listen bucket notification is applicable for this layer.
```

Comment:
xlObjects is not compatible with ObjectInterface and hence you need not add this function here

Reply:
functions like savePolicyConfig take ObjectLayer as parameter, hence the need for xlObjects to implement this function.